articles/other/other-how-delete-sso-user.md

```diff
@@ -0,0 +1,190 @@
+---
+title: How to delete single sign on users | UKCloud Ltd
+description: Describes the process for deleting single sign on (SSO) users
+services: other
+author: Sue Highmoor
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Delete SSO users
+toc_fullpath: How To/other-how-delete-sso-user.md
+toc_mdlink: other-how-delete-sso-user.md
+---
+
+# How to delete single sign on users
```

Comment:
I'd prefer use of 'Single Sign-On', both here and more widely across the site wherever possible.

Reply:
Fixed